<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial</class>
 <widget class="QMainWindow" name="Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comunicação serial</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbHard">
      <property name="title">
       <string>Configuração de hardware:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lbPort">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbBaud">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Baud rate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="cbPort"/>
           </item>
           <item>
            <widget class="QComboBox" name="cbBaud"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lbDtBit">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Data bits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbPar">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Paridade</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QComboBox" name="cbDtBit"/>
           </item>
           <item>
            <widget class="QComboBox" name="cbPar"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="lbStop">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Stop bit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbFlow">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Controle de fluxo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QComboBox" name="cbStop"/>
           </item>
           <item>
            <widget class="QComboBox" name="cbFlow"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="pteMoni">
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit {
    background-color: #121212;      /* Fundo quase preto (estilo Dark Mode) */
    color: #00FF00;                 /* Texto verde &quot;Matrix/Terminal&quot; para destaque */
    border: 2px solid #333333;      /* Borda discreta */
    border-radius: 4px;
    font-family: 'Consolas', 'Monospace', 'Courier New'; /* Fontes monoespaçadas */
    font-size: 10pt;
    padding: 5px;
}

/* Estilizando a barra de rolagem para combinar */
QScrollBar:vertical {
    border: none;
    background: #222222;
    width: 10px;
}
QScrollBar::handle:vertical {
    background: #444444;
    min-height: 20px;
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbStatus">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbMain">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbRefresh">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
